cf/i18n/resources/de-de.all.json

```diff
@@ -1349,11 +1349,11 @@
   },
   {
     "id": "Custom buildpack by name (e.g. my-buildpack) or Git URL (e.g. 'https://github.com/cloudfoundry/java-buildpack.git') or Git URL with a branch or tag (e.g. 'https://github.com/cloudfoundry/java-buildpack.git#v3.3.0' for 'v3.3.0' tag). To use built-in buildpacks only, specify 'default' or 'null'",
-    "translation": "Angepasstes Buildpack nach Name (z. B. my-buildpack) oder Git-URL (z. B. 'https://github.com/cloudfoundry/java-buildpack.git') oder Git-URL mit Zweig oder Tag (z. B. 'https://github.com/cloudfoundry/java-buildpack.git#v3.3.0' für Tag 'v3.3.0'). Geben Sie zur ausschließlichen Verwendung von integrierten Buildpacks 'default' oder 'null' an."
+    "translation": "Angepasstes Buildpack nach Name (z. B. my-buildpack) oder Git-URL (z. B. 'https://github.com/cloudfoundry/java-buildpack.git') oder Git-URL mit Zweig oder Tag (z. B. 'https://github.com/cloudfoundry/java-buildpack.git#v3.3.0' für Tag 'v3.3.0'). Geben Sie zur ausschließlichen Verwendung von integrierten Buildpacks 'default' oder 'null' an"
   },
```

Comment:
The German person on my team (and some googling I did) seem to indicate it's more common to not use a space in `z.B.`